.github/workflows/deployment.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   deploy:
     name: Build and deploy project
     runs-on: ubuntu-latest
+    environment: Staging
```

Comment:
What do we need this for?

Reply:
We don't anymore. It was for setting the secrets from github. I'll remove it.

Reply:
Cleaned up: https://github.com/gipfeli-io/gipfeli-api/pull/16